### PR TITLE
Fix comparison between signed and unsigned integer expressions in ten…

### PR DESCRIPTION
**Pull request by Kh4L**
…sor_format.h

### CODE REVIEW

File: tensorflow/core/framework/dataset.h [hunk 1]
```diff
@@ -162,7 +162,7 @@ class GraphDefBuilderWrapper {
                     const std::vector<std::pair<StringPiece, AttrValue>>& attrs,
                     Node** output) {
     std::vector<std::pair<size_t, Node*>> enumerated_inputs(inputs.size());
-    for (int i = 0; i < inputs.size(); i++) {
+    for (int i = 0; static_cast<size_t>(i) < inputs.size(); i++) {
```
Comment (mrry) [inline tensorflow/core/framework/dataset.h]: Let's use the following simpler code here:

```suggestion
    for (size_t i = 0; i < inputs.size(); i++) {
```

**Review by mrry — CHANGES_REQUESTED** [review]

File: tensorflow/core/framework/dataset.h (hunk 1 above)
Comment (Kh4L) [reply tensorflow/core/framework/dataset.h]: Sure, that makes more sense :smile:

**Review by mrry — DISMISSED** [review]
Thanks for the cleanup!

**Comment by mrry** [comment]
The clang-format errors need to be fixed first. The Ubuntu and Window GPU failures look like flakes.

**Comment by harshini-gadige** [comment]
> The clang-format errors need to be fixed first. The Ubuntu and Window GPU failures look like flakes.

@Kh4L  Could you run clang-format-3.6.0 on this?

**Comment by harshini-gadige** [comment]
> @harshini-gadige done

@mrry Can you please check now and approve ?

**Review by mrry — APPROVED** [review]

**Pull request merged**